CMakeLists.txt

```diff
@@ -1316,19 +1310,19 @@ IF(BUILD_TESTING)
     target_link_libraries(SixTestWrapper ${MSYS_QUADMATH})
     target_link_libraries(readDump3 ${MSYS_QUADMATH})
   endif()
-ENDIF()
+endif()
 
 #To make packages, run "make package" in the build directory
-#SET(CPACK_GENERATOR RPM DEB STGZ TGZ TBZ2 TXZ ZIP 7Z)
-SET(CPACK_GENERATOR RPM DEB TGZ ZIP) #Take only the most common targets
-#SET(CPACK_PACKAGE_CONTACT "sixtrack.support@cern.ch") # non-existent email address
-SET(CPACK_PACKAGE_VERSION ${SixTrack_VERSION})
-SET(CPACK_PACKAGE_VENDOR "CERN")
-SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "SixTrack symplectic particle tracking code for particle accelerators.")
-SET(CPACK_PACKAGE_VERSION_MAJOR ${SixTrack_VERSION_MAJOR})
-SET(CPACK_PACKAGE_VERSION_MINOR ${SixTrack_VERSION_MINOR})
-SET(CPACK_PACKAGE_VERSION_PATCH ${SixTrack_VERSION_PATCH})
-INCLUDE(CPack)
+#set(CPACK_GENERATOR RPM DEB STGZ TGZ TBZ2 TXZ ZIP 7Z)
+set(CPACK_GENERATOR RPM DEB TGZ ZIP) #Take only the most common targets
+#set(CPACK_PACKAGE_CONTACT "sixtrack.support@cern.ch") # non-existent email address
+set(CPACK_PACKAGE_VERSION ${SixTrack_VERSION})
+set(CPACK_PACKAGE_VENDOR "CERN")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "SixTrack symplectic particle tracking code for particle accelerators.")
+set(CPACK_PACKAGE_VERSION_MAJOR ${${PROJECT_NAME}_VERSION_MAJOR})
```

Comment:
why ${PROJECT_NAME} is  used here only?

Reply:
Should be used for line 1319 too I see, but this bit of code doesn't work anyway.

Reply:
Works for me (mostly).

Reply:
well one line needs uncommenting for the .deb generation to work.

Reply:
Yeah, I couldn't get the deb one to work properly, but I checked that it sets package version correctly anyway.

Reply:
Which line is upsetting deb creation?

Reply:
CPackDeb: Debian package requires a maintainer for a package, set
  CPACK_PACKAGE_CONTACT or CPACK_DEBIAN_PACKAGE_MAINTAINER

Reply:
We can use  sixtrack.contact@cern.ch or sixtrack.support@cern.ch. The emails are redirected to my account.